test/plug.test.ts

```diff
@@ -236,7 +317,10 @@ describe('The Croct plug', () => {
         croct.extend('foo', fooFactory);
         croct.extend('bar', barFactory);
 
-        const config: SdkFacadeConfiguration = {appId: APP_ID};
+        const config: SdkFacadeConfiguration = {
+            appId: APP_ID,
+            test: false,
+        };
```

Comment:
Why this change? The `beforeEach` should cause the test mode to be false on our existing tests